### PR TITLE
Update dataset to 0.2

### PR DESCRIPTION
**Pull request by daemon**
- Add 24 query--document pairs
- Remove duplicated example

### CODE REVIEW

**Comment by daemon** [comment]
Closes #22.

**Comment by lintool** [comment]
@rodrigonogueira4 @ronakice take a look please?

File: pygaggle/data/msmarco.py [hunk 1]
```diff
@@ -131,14 +129,11 @@ def to_relevance_examples(self,
             mean_stats['Existing MRR'].append(1 / (ex_index + 1)
                                               if ex_index < len(ex.candidates)
                                               else 0)
-            mean_stats['Existing MRR@10'].append(1 / (ex_index + 1)
-                                                 if ex_index < 10 else 0)
+            mean_stats['Existing MRR@10'].append(1 / (ex_index + 1) if ex_index < 10 else 0)
```
Comment (ronakice) [inline pygaggle/data/msmarco.py:132]: Won't switching make flake8 angry? Unless we plan to go with a different line length than 80, in which case this is fine!

File: pygaggle/data/kaggle.py [hunk 2]
```diff
@@ -82,10 +82,8 @@ def to_senticized_dataset(self,
             mean_stats['Random P@1'].append(np.mean(int_rels))
             n = len(int_rels) - p
             N = len(int_rels)
-            mean_stats['Random R@3'].append(1 - (n * (n - 1) * (n - 2)) / (N *
-                                            (N - 1) * (N - 2)))
-            numer = np.array([sp.comb(n, i) / (N - i)
-                              for i in range(0, n + 1)]) * p
+            mean_stats['Random R@3'].append(1 - (n * (n - 1) * (n - 2)) / (N * (N - 1) * (N - 2)))
+            numer = np.array([sp.comb(n, i) / (N - i) for i in range(0, n + 1)]) * p
```
Comment (ronakice) [inline pygaggle/data/kaggle.py:86]: Same here

File: pygaggle/data/msmarco.py [hunk 3]
```diff
@@ -107,15 +107,13 @@ def to_relevance_examples(self,
         mean_stats = defaultdict(list)
         for ex in self.examples:
             int_rels = np.array(list(map(int, example_map[ex.qid][3])))
-            p = int_rels.sum()/(len(ex.candidates) - 1) if is_duo \
-                else int_rels.sum()
+            p = int_rels.sum()/(len(ex.candidates) - 1) if is_duo else int_rels.sum()
```
Comment (ronakice) [inline pygaggle/data/msmarco.py:110]: Same!

File: pygaggle/data/msmarco.py [hunk 4]
```diff
@@ -107,15 +107,13 @@ def to_relevance_examples(self,
...
             mean_stats['Random P@1'].append(np.mean(int_rels))
             n = len(ex.candidates) - p
             N = len(ex.candidates)
             if len(ex.candidates) <= 1000:
                 mean_stats['Random R@1000'].append(1 if 1 in int_rels else 0)
-            numer = np.array([sp.comb(n, i) / (N - i) for i in range(0, n + 1)
-                             if i != N]) * p
+            numer = np.array([sp.comb(n, i) / (N - i) for i in range(0, n + 1) if i != N]) * p
```
Comment (ronakice) [inline pygaggle/data/msmarco.py:116]: Same

File: pygaggle/data/msmarco.py (hunk 1 above)
Comment (daemon) [reply pygaggle/data/msmarco.py:132]: Yeah, we've decided to go with 100.

**Pull request merged**